reference/5.0/CimCmdlets/Register-CimIndicationEvent.md

```diff
@@ -265,67 +289,65 @@ Accept wildcard characters: False
 ```
 
 ### -Query
-Specifies a query to run on the CIM server.
-
-You can specify the query dialect using the QueryDialect parameter.
 
-If the value specified contains double quotes ("), single quotes ('), or a backslash (\\), you must escape those characters by prefixing them with the backslash (\\) character.
-If the value specified uses the WQL LIKE operator, then you must escape the following characters by enclosing them in square brackets (\[\]): percent (%), underscore (_), or opening square bracket (\[).
+Specifies a query to run on the CIM server. If the value specified contains double quotes ("),
+single quotes ('), or a backslash (\\), you must escape those characters by prefixing them with the
+backslash (\\) character. If the value specified uses the WQL LIKE operator, then you must escape
+the following characters by enclosing them in square brackets (\[\]): percent (%), underscore (_),
```

Comment:
I think you need to markdown escape the underscore or remaining content is in italics.

Reply:
fixed